Mathlib/Analysis/Meromorphic/Order.lean

```diff
@@ -132,6 +132,26 @@ theorem order_mul {f g : 𝕜 → 𝕜} {x : 𝕜} (hf : MeromorphicAt f x) (hg
     (hf.mul hg).order = hf.order + hg.order :=
   hf.order_smul hg
 
+/-- The order of the inverse is the negative of the order. -/
+theorem order_inv {f : 𝕜 → 𝕜} {z₀ : 𝕜} (hf : MeromorphicAt f z₀) :
+    hf.order = -hf.inv.order := by
```

Comment:
I would expect the following statement:
```suggestion
theorem order_inv {f : 𝕜 → 𝕜} {z₀ : 𝕜} (hf : MeromorphicAt f z₀) :
    hf.inv.order = -hf.order := by
```
If you agree, could you please adjust the proof accordingly?

Reply:
Implemented.